pkg/proxy/metrics/metrics.go

```diff
@@ -263,7 +273,7 @@ var (
 			Help:           "Number of services with a Local traffic policy and no endpoints",
 			StabilityLevel: metrics.ALPHA,
 		},
-		[]string{"traffic_policy"},
+		[]string{"traffic_policy", "ipFamily"},
```

Comment:
@aojea Should I use `ip_family` instead?
Looks like we are using snake casing for metrics name and dimensions.

Reply:
I think you are right Daman, but @richabanker @dgrisonnet can you please confirm us what is the right name for the label?